### DISCUSSION TITLE
Filters loses focus when try to filter

### DISCUSSION THREAD

**Discussion by rohit-squirrel** [Comcast/react-data-grid#3282] [General]
When ever trying to filter it loses focus, I am using headerRenderer to render filters and when type in anything to filters it loses focus.

Data grid version : 7.0.0-canary.30
react version : 17

**Comment by godon019** [comment]
that is an old version of RDG and I don't think it supports react 17. I would recommend you to update packages

**Reply by rohit-squirrel** [reply]
Okay i have updated the package to the latest and now there is different issue that i am facing, now columns are not populating as it should be.
It is now taking columns headers as rows so it is showing only one column.
 Can you help why this could be happening?

data grid version: 7.0.0-beta.34
react version : 18

**Reply by godon019** [reply]
could you provide a reproduction repo of this issue?
